alibi_detect/utils/pytorch/kernels.py

```diff
@@ -88,6 +168,271 @@ def forward(self, x: Union[np.ndarray, torch.Tensor], y: Union[np.ndarray, torch
         return kernel_mat.mean(dim=0)  # [Nx, Ny]
 
 
+class RationalQuadratic(BaseKernel):
+    def __init__(
+        self,
+        alpha: torch.Tensor = None,
+        init_fn_alpha: Callable = pseudo_init_fn,
+        sigma: torch.Tensor = None,
```

Comment:
Is this a sensible initialisation function to have as default? The name `pseudo` makes me think perhaps not? If so would be worth elaborating in the docstring of `pseudo_init_fn`. If not we should raise a warning or some other safeguard.

Reply:
Now removed given the new parameter implementation.